rules/show-certification-award/rule.md

```diff
@@ -0,0 +1,39 @@
+---
+type: rule
+title: Do you show Certification Reward?
+uri: show-certification-award
+authors:
+  - title: Stephan Fako
+    url: https://www.ssw.com.au/people/stephan-fako/
+related: []
+created: 2024-04-08T06:55:48.587Z
+guid: 94ab00f4-a6d5-4e8f-aca3-68d6cc0a3498
+---
+**Do you show Certification Reward?**
+ 
+Encouraging professional development and recognizing achievements are crucial for fostering a positive workplace culture. SSW's Certification Reward system is a testament to this, offering special leave to employees who complete Microsoft certifications. This initiative not only motivates employees to pursue further learning but also highlights the company's commitment to staying at the forefront of technology.
+ 
+---
```

Comment:
Hi @stephanfakossw 

- [ ] 1. Can you replace this `---` with `<!--endintro-->` 

This is to set the blurb (intro) for the rule as per number 8 in https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Create-Rules#manual-creation-in-github